packages/expo-doctor/src/utils/getRemoteVersionsForSdkAsync.ts

```diff
@@ -0,0 +1,26 @@
+import { getVersionsAsync } from '../api/getVersionsAsync';
+
+export type DependencyList = Record<string, string>;
+
+export const getRemoteVersionsForSdkAsync = async (
+  sdkVersion?: string
+): Promise<DependencyList> => {
+  const { sdkVersions } = await getVersionsAsync({ skipCache: true });
+  if (sdkVersion && sdkVersion in sdkVersions) {
+    const { relatedPackages, facebookReactVersion, facebookReactNativeVersion } = sdkVersions[
+      sdkVersion
+    ];
+    const reactVersion = facebookReactVersion
+      ? {
+          react: facebookReactVersion,
+          'react-dom': facebookReactVersion,
+        }
+      : undefined;
```

Comment:
This field exists since SDK 14, I think it's safe to simplify this like facebookReactNativeVersion below.